### PR TITLE
Bug Fix MS-236

### PR DESCRIPTION
**Pull request by GulshanBaryah**

### CODE REVIEW

File: DFC.App.MatchSkills.Test/Helpers/MockHelpers.cs [hunk 1]
```diff
@@ -5,14 +5,25 @@
...
+using NSubstitute;
+using FluentAssertions.Common;
+using Microsoft.AspNetCore.Mvc;
 
 namespace DFC.App.MatchSkills.Test.Helpers
 {
     public static class MockHelpers
     {
-        public static  Mock<HttpMessageHandler> GetMockMessageHandler(string contentToReturn="{'Id':1,'Value':'1'}", HttpStatusCode statusToReturn=HttpStatusCode.OK)
+        public static string GetCookieName { get; } = ".matchSkill-session";
```
Comment (syatessoftware) [inline DFC.App.MatchSkills.Test/Helpers/MockHelpers.cs]: Can You reference the Static CookieName from CookieService please

File: DFC.App.MatchSkills.Test/Unit/Controllers/OccupationSearchControllerTests.cs [hunk 2]
```diff
@@ -67,19 +67,54 @@ public void  When_OccupationSearch_Then_ShouldReturnOccupations()
...
             var occupations =   sut.OccupationSearchAuto("Renewable");
             
             occupations.Result.Should().NotBeNull();
         }
 
-       
+        [Test]
+        public void  When_GetOccupationIdFromName_Then_ShouldReturnOccupationId()
+        {
+            const string skillsJson ="{\"occupations\": [{\"uri\": \"http://data.europa.eu/esco/occupation/114e1eff-215e-47df-8e10-45a5b72f8197\",\"occupation\": \"Renewable energy consultant\",\"alternativeLabels\": [\"alt 1\"],\"lastModified\": \"03-12-2019 00:00:01\"}]}";
```
Comment (syatessoftware) [inline DFC.App.MatchSkills.Test/Unit/Controllers/OccupationSearchControllerTests.cs]: Seems like we're using the const, multiple times, can we instantiate it at the top of the class instead please?

Comment (GulshanBaryah) [reply DFC.App.MatchSkills.Test/Unit/Controllers/OccupationSearchControllerTests.cs]: Done

File: DFC.App.MatchSkills.Test/Helpers/MockHelpers.cs (hunk 1 above)
Comment (GulshanBaryah) [reply DFC.App.MatchSkills.Test/Helpers/MockHelpers.cs]: Done

**Review by syatessoftware — APPROVED** [review]

**Pull request merged**